spec/memo_wise_spec.rb

```diff
@@ -396,6 +420,24 @@ def no_args
         expect(instance.with_keyword_args_counter).to eq(3)
       end
 
+      it "resets memoization for methods with positional and keyword args" do
+        instance.with_positional_and_keyword_args(1, b: 2)
+        instance.with_positional_and_keyword_args(2, b: 3)
+        instance.reset_memo_wise(:with_positional_and_keyword_args, 1, b: 2)
+
+        expect(Array.new(4) do
+          instance.with_positional_and_keyword_args(1, b: 2)
+        end).to all eq("with_positional_and_keyword_args: a=1, b=2")
+
+        expect(Array.new(4) do
+          instance.with_positional_and_keyword_args(2, b: 3)
+        end).to all eq("with_positional_and_keyword_args: a=2, b=3")
+
+        # This should be executed twice for each set of arguments passed,
```

Comment:
This comment is a little confusing—it's really once for each set of arguments passed, which is twice total. Could you rephrase this?